src/System.Management.Automation/FormatAndOutput/common/PSStyle.cs

```diff
@@ -34,6 +36,17 @@ public enum ProgressView
         /// <summary>Classic rendering of progress.</summary>
         Classic = 1,
     }
+    
+    /// <summary>
+    /// Type is used for custom formatting.
+    /// </summary>
+    public class AnsiDictionary : Dictionary<string, string>
```

Comment:
Should this class be sealed?  Also, is this used anywhere?

Reply:
Actually, I think I intended it for something, but it's actually not used.  Will remove.